test/exponential_backoff_test.rb

```diff
@@ -2,7 +2,7 @@
 
 class ExponentialBackoffTest < Minitest::Test
   def setup
-    Resque.redis.flushall
+    Resque.redis.redis.flushall
```

Comment:
delegating missing methods to redis::client's redis attr_reader is deprecated, do the delegation manually to avoid warnings